cryptostock/account/tests/test_jwt_auth.py

```diff
@@ -0,0 +1,43 @@
+from account.tests.conftest import TEST_USERS_PASSWORDS
+from utils.jwt_views import MyTokenObtainPairSerializer
+
+
+def test_jwt_auth(user_one, auth_broker):
+    data = {"username": user_one.username, "password": TEST_USERS_PASSWORDS["user_one"]}
+    auth_broker.logout()
+
+    response = auth_broker.post("/api/auth/", data=data)
+    token = MyTokenObtainPairSerializer.get_token(user_one)
+
+    assert response.status_code == 200
+    assert "refresh" and "access" in response.json().keys()
```

Comment:
Try in console:
```suggestion
     "refresh" and "access" in {"access": 123}
```
What does it return?

Reply:
fixed